tests/integration/records/test_views.py

```diff
@@ -19,6 +19,7 @@ def test_literature_search_application_json_get(
 ):
     data = {
         "control_number": 666,
+        "document_type": ["article"],
```

Comment:
it's automatically added as well as ``_collections``

Reply:
I don't want to rely on "automatically" edition, when it changes it will break the test, but feature would be already working.